app/Models/Team.php

```diff
@@ -32,4 +32,16 @@ public function users()
         // 中間テーブル名を明記（デフォルトは'team_user'で探しちゃう）←学習のため
         return $this->belongsToMany(User::class, 'members');
     }
+
+    public function isManager(User $user)
+    {
+        // 受け取ったUserがマネージャーをしているチームかどうか判定（←学習のため）
+        // Membersにレコードが存在するかチェック
+        $isTeam = $user->members()
```

Comment:
もしも一時変数にするのであれば、`$isManager` で良さそうです。

また、あえて変数に格納せずに returnしてしまうのもできそうですね。その場合には45行目の削除をお忘れなく

```suggestion
        return $user->members()
```